pipeline/transformer/transformer_tei/src/main/scala/weco/pipeline/transformer/tei/transformers/TeiNotes.scala

```diff
@@ -0,0 +1,35 @@
+package weco.pipeline.transformer.tei.transformers
+
+import weco.catalogue.internal_model.work.{Note, NoteType}
+import weco.pipeline.transformer.tei.NormaliseText
+
+import scala.xml.{Elem, NodeSeq}
+
+object TeiNotes {
+  def apply(xml: Elem): List[Note] =
+    apply(xml \\ "msDesc" \ "msContents")
+
+  def apply(node: NodeSeq): List[Note] =
+    getColophon(node).toList
+
+  /** The colophon is in `colophon` nodes under `msContents` or `msItem`.
+    *
+    * <TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id={id}>
+    *   <teiHeader>
+    *     <fileDesc>
+    *       <sourceDesc>
+    *         <msDesc xml:lang="en" xml:id="MS_Arabic_1">
+    *           <msContents>
+    *             <colophon> <locus>F. 9v</locus> </colophon>
+    *
+    */
+  private def getColophon(value: NodeSeq): Seq[Note] =
+    (value \ "colophon")
+      .map { n =>
+        n.text.trim
```

Comment:
I _think_ that if you use `.text` you shouldn't get comments? Have you tried to see if that tests passes even without the strip comments function?

Reply:
Oh yeah, it totally does. My bad, fixed, thanks. 🙈